examples/collector-exporter-node/docker/docker-compose.yaml

```diff
@@ -5,8 +5,6 @@ services:
     image: otel/opentelemetry-collector:latest
 #    image: otel/opentelemetry-collector:0.6.0
     command: ["--config=/conf/collector-config.yaml", "--log-level=DEBUG"]
-    networks:
```

Comment:
the networks was needed for tracer, otherwise the collector was complaining about it, have you test that traces are still working ?

Reply:
The only way I was able to get prometheus to work was to remove networks. It still works for traces without any warnings/errors.

Reply:
hmm maybe something has changed again in latest version so now the networks is not needed - I have removed this once, then it wasn't working after upgrade to 0.6 so I added again, now it is removed but if it is working again then fine :).